tools/astarte_e2e/lib/astarte_e2e/config.ex

```diff
@@ -113,6 +116,50 @@ defmodule AstarteE2E.Config do
     type: :pos_integer,
     default: 10
 
+  @envdoc "The comma-separated email recipients."
+  app_env :mailer_to_address, :astarte_e2e, :mailer_to_address,
+    os_env: "ASTARTE_E2E_MAILER_TO_ADDRESS",
+    type: ListOfStrings,
+    default: ""
+
+  @envdoc "The notification email sender."
+  app_env :mailer_from_address, :astarte_e2e, :mailer_from_address,
+    os_env: "ASTARTE_E2E_MAILER_FROM_ADDRESS",
+    type: :binary,
+    default: ""
+
+  @envdoc """
+  The mail service's API key. Currently only Mailgun is supported. This env var must be set and
+  valid to use the mail service.
+  """
+  app_env :mail_api_key, :astarte_e2e, :mail_api_key,
+    os_env: "ASTARTE_E2E_MAIL_API_KEY",
+    type: :binary
+
+  @envdoc """
+  The mail domain. Currently only Mailgun is supported. This env var must be set and
```

Comment:
No need to repeat this over and over: `The mail API base URI. Currently only Mailgun is supported.`